### PR TITLE
Add Pre-releases filter in `setup-protoc` action

### PR DESCRIPTION
**Pull request by rsora**
This PR modifies the `setup-protoc` action in order to:
- Fetch available tags for `protobuf` repo using https://api.github.com/repos/protocolbuffers/protobuf/releases instead of using refs
- Filter out pre-release tags by specifying the new `include-pre-releases` flag in the action configuration (default is `false`)

Having by default `include-pre-releases` set to `false`  solves a bug caused by a `protobuf` RC release that has malformed download urls, not respecting the semver notation in the filename (i.e. [v3.10.0-rc1](https://github.com/protocolbuffers/protobuf/releases/tag/v3.10.0-rc1) )

### CODE REVIEW

File: setup-protoc/src/installer.ts [hunk 1]
```diff
@@ -208,3 +212,15 @@ function normalizeVersion(version: string): string {
...
+function includePrerelease(
+  isPrerelease: boolean,
+  includePrereleases: boolean
+): boolean {
+  if (!includePrereleases) {
+    if (isPrerelease) {
+      return false;
+    }
+  }
+  return true;
```
Comment (masci) [inline setup-protoc/src/installer.ts]: this could be `return includePrereleases || !isPrerelease`

Comment (rsora) [reply setup-protoc/src/installer.ts]: I thought a function could be more explicit, but we can replace it with boolean logic + comment :+1:

Comment (masci) [reply setup-protoc/src/installer.ts]: I usually prefer explicit code as well but in this case the one-liner is pretty readable IMO

File: setup-protoc/node_modules/@actions/core/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "_args": [
     [
       "@actions/core@1.0.0",
-      "/home/massi/dev/actions/setup-protoc"
+      "/home/rsora/code/projects/arduino/actions/setup-protoc"
```
[the remark this answers is not in the document]
Comment (rsora) [reply setup-protoc/node_modules/@actions/core/package.json]: AFAIK pushing `node_modules` is non a best practice :smile: . 
I checked for example [coveralls](https://github.com/coverallsapp/github-action) and they have  [hardcoded paths](https://github.com/coverallsapp/github-action/blob/8d2940685b011d380ca033e2786e92812f915b1d/node_modules/%40actions/core/package.json#L10 ) as well

**Pull request merged**